### ISSUE TITLE
Document the meaning and units of measurement of all the parameters.

### ISSUE THREAD

**Issue by axch** [edoddridge/aronnax#58]
For now, expanding the comments in the top-level `parameters.in` file is an adequate place to capture this knowledge.  I believe Fortran namelist input permits interspersing line comments with the entries of the namelist, and may even permit comments at the end of a line, like
```
 &NUMERICS
 au = 200.,  ! viscosity, in Pa * s
 ah = 0.0,   ! thickness diffusivity, in m^2/s
 ...
```

**Comment by edoddridge** [comment]
This was closed by #135, which added a new config file to the base directory of the repo that included comments describing each of the parameters, and displayed this file in the documentation.

**Comment by axch** [comment]
Just a couple little nits from looking that file over:
- What are the units in which the Coriolis parameter is given?
- Can you give the bit meanings for the wetmask (1 for water and 0 for land?)
